### ISSUE TITLE
fixturenet-eth-geth container build fails

### ISSUE THREAD

**Issue by dboreham** [cerc-io/stack-orchestrator#397]
Probably to do with a base image having updated to bundle Python 3.11:

```
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
Removing intermediate container 65b600758b36
 ---> 2f7e3a39c1dc
Step 12/17 : COPY genesis /opt/testnet
 ---> a0a9e65c626f
Step 13/17 : COPY run-el.sh /opt/testnet/run.sh
 ---> a003b75698d5
Step 14/17 : RUN cd /opt/testnet && make genesis-el
 ---> Running in 229f669d4d30
cd el; ./build_el.sh
Traceback (most recent call last):
  File "/apps/el-gen/genesis_geth.py", line 1, in <module>
    from web3.auto import w3
  File "/usr/lib/python3.11/site-packages/web3/__init__.py", line 6, in <module>
    from eth_account import (
  File "/usr/lib/python3.11/site-packages/eth_account/__init__.py", line 1, in <module>
    from eth_account.account import (
  File "/usr/lib/python3.11/site-packages/eth_account/account.py", line 59, in <module>
    from eth_account.messages import (
  File "/usr/lib/python3.11/site-packages/eth_account/messages.py", line 26, in <module>
    from eth_account._utils.structured_data.hashing import (
  File "/usr/lib/python3.11/site-packages/eth_account/_utils/structured_data/hashing.py", line 9, in <module>
    from eth_abi import (
  File "/usr/lib/python3.11/site-packages/eth_abi/__init__.py", line 6, in <module>
    from eth_abi.abi import (  # NOQA
  File "/usr/lib/python3.11/site-packages/eth_abi/abi.py", line 1, in <module>
    from eth_abi.codec import (
  File "/usr/lib/python3.11/site-packages/eth_abi/codec.py", line 16, in <module>
    from eth_abi.decoding import (
  File "/usr/lib/python3.11/site-packages/eth_abi/decoding.py", line 14, in <module>
    from eth_abi.base import (
  File "/usr/lib/python3.11/site-packages/eth_abi/base.py", line 7, in <module>
    from .grammar import (
  File "/usr/lib/python3.11/site-packages/eth_abi/grammar.py", line 4, in <module>
    import parsimonious
  File "/usr/lib/python3.11/site-packages/parsimonious/__init__.py", line 9, in <module>
    from parsimonious.grammar import Grammar, TokenGrammar
  File "/usr/lib/python3.11/site-packages/parsimonious/grammar.py", line 14, in <module>
    from parsimonious.expressions import (Literal, Regex, Sequence, OneOf,
  File "/usr/lib/python3.11/site-packages/parsimonious/expressions.py", line 9, in <module>
    from inspect import getargspec
ImportError: cannot import name 'getargspec' from 'inspect' (/usr/lib/python3.11/inspect.py)
Traceback (most recent call last):
  File "/opt/testnet/el/../accounts/mnemonic_to_csv.py", line 1, in <module>
    from web3.auto import w3
  File "/usr/lib/python3.11/site-packages/web3/__init__.py", line 6, in <module>
    from eth_account import (
  File "/usr/lib/python3.11/site-packages/eth_account/__init__.py", line 1, in <module>
    from eth_account.account import (
  File "/usr/lib/python3.11/site-packages/eth_account/account.py", line 59, in <module>
    from eth_account.messages import (
  File "/usr/lib/python3.11/site-packages/eth_account/messages.py", line 26, in <module>
    from eth_account._utils.structured_data.hashing import (
  File "/usr/lib/python3.11/site-packages/eth_account/_utils/structured_data/hashing.py", line 9, in <module>
    from eth_abi import (
  File "/usr/lib/python3.11/site-packages/eth_abi/__init__.py", line 6, in <module>
    from eth_abi.abi import (  # NOQA
  File "/usr/lib/python3.11/site-packages/eth_abi/abi.py", line 1, in <module>
    from eth_abi.codec import (
  File "/usr/lib/python3.11/site-packages/eth_abi/codec.py", line 16, in <module>
    from eth_abi.decoding import (
  File "/usr/lib/python3.11/site-packages/eth_abi/decoding.py", line 14, in <module>
    from eth_abi.base import (
  File "/usr/lib/python3.11/site-packages/eth_abi/base.py", line 7, in <module>
    from .grammar import (
  File "/usr/lib/python3.11/site-packages/eth_abi/grammar.py", line 4, in <module>
    import parsimonious
  File "/usr/lib/python3.11/site-packages/parsimonious/__init__.py", line 9, in <module>
    from parsimonious.grammar import Grammar, TokenGrammar
  File "/usr/lib/python3.11/site-packages/parsimonious/grammar.py", line 14, in <module>
    from parsimonious.expressions import (Literal, Regex, Sequence, OneOf,
  File "/usr/lib/python3.11/site-packages/parsimonious/expressions.py", line 9, in <module>
    from inspect import getargspec
ImportError: cannot import name 'getargspec' from 'inspect' (/usr/lib/python3.11/inspect.py)
make: *** [Makefile:9: genesis-el] Error 1
The command '/bin/sh -c cd /opt/testnet && make genesis-el' returned a non-zero code: 2
Error running build for cerc/fixturenet-eth-geth
FATAL Error: container build failed and --continue-on-error not set, exiting
root@ubuntu-c-4-8gib-tor1-01:~# client_loop: send disconnect: Broken pipe
```

**Comment by dboreham** [comment]
Problem seems to come from here: https://github.com/cerc-io/stack-orchestrator/blob/main/app/data/container-build/cerc-fixturenet-eth-geth/Dockerfile#L9
which now gets us Python 3.11:

```
(63/78) Installing python3 (3.11.3-r11)
```

**Comment by telackey** [comment]
Initial fix is to peg the alpine version to 3.17, but the more complete fix is to get us on a current version of ethereum-genesis-generator.  That will be involved, because it looks like the project has refactored a fair amount.

https://github.com/cerc-io/stack-orchestrator/commit/aa702737eff5762882232d0e1f1ac589bdf10481
